plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnector.java

```diff
@@ -47,7 +47,7 @@
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
-public class TestPostgreSqlConnectorTest
+public class TestPostgreSqlConnector
```

Comment:
The TestPostgreSqlConnectorTest name was intentional and deliberate.

Reply:
I know. I wanted to challenge this.